src/views/Generator/GeneratorTabs/ScriptPreview.tsx

```diff
@@ -18,27 +26,32 @@ export function ScriptPreview() {
 
   return (
     <Flex direction="column" height="100%">
-      <Flex p="2" gap="2" justify="end">
-        <Button
+      <Flex py="1" px="2" gap="2" align="center" justify="end">
+        <GhostButton
```

Comment:
I changed these to ghost buttons because

1. it makes the run in cloud button more prominent
2. the soft variant just doesn't look good